_posts/2015-09-xx-creating-systems-with-pipelining.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "creating systems with pipelining"
+date: "2015-09-xx"
+icon: wrench
+tags: pipelines, virtual mashines, development systems
+categories: Application Management
+authors: ["Carsten", "Karsten"]
+---
+
+Today I will introduce some special topic to our readers, called [pipelining](https://en.wikipedia.org/wiki/Pipeline_(software)). When we at ePages speak about pipelining, we understand it as an automatic workflow, integrating many steps using different tools to reach a specific goal. The goal we had was to create a simple way to provide a highly specialized system. This system can be for development, external or even live.
```

Comment:
Remove the first sentence.
Rephrase second sentence to: If we talk about [pipelining]((https://en.wikipedia.org/wiki/Pipeline_(software)) at ePages, we understand it as an automatic workflow. We are integrating many steps using different tools to reach a specific goal: creating a simple way to provide a highly specialised system.

> This is kind of an introduction or "teaser" for the reader.

You can than start a new paragraph with this: This system can be used for development purposes, external usage or even live deployment.